src/services/billing/cost-management/widgets/CostTrendLine.vue

```diff
@@ -303,6 +303,7 @@ export default {
             state.items = await listCostAnalysisData(period, filters);
             const { slicedChartData, slicedLegends } = await getSlicedChartDataAndLegends();
             state.chartData = slicedChartData;
+            if (state.items.length === 0) emit('rendered');
```

Comment:
```suggestion
            if (state.chartData.length === 0) emit('rendered');
```

여긴 `chartData` 로 확인해야 하지 않을까요?

Reply:
items를 기반으로 임의로 slice하여 chartData를 만드니까 괜찮지않을까요???

Reply:
제 의견은
`getSlicedChartDataAndLegends()`  가 어떤 일을 수행하는지, 
그로 인해 리턴된 `slicedChartData`는 `items`와 정확히 length가 동일한 배열이 리턴되는 것인지 알 수 없으므로,
`slicedChartData`의 length를 확인하는게 더 확실해보입니다!

Reply:
그렇다면 차라리 순서를 바꾸는게 맞다고 생각해요! 좀 더 위로올리거나. items를 체크해야하는 의견에 동의합니다!